package.json

```
@@ -13,9 +13,10 @@
     ]
   },
   "engines": {
-    "node": ">=14.18.1",
-    "npm": "please-use-yarn",
-    "yarn": ">=1.22.0 <2"
+    "npm": "please-use-pnpm",
+    "yarn": "please-use-pnpm",
+    "node": "16",
+    "pnpm": ">=7.28"
```

```suggestion
    "pnpm": ">=7.28 <18"
```
Will want to limit this since PNPM 8 has breaking changes